libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -206,61 +186,23 @@ public void onCreate(Bundle savedInstanceState) {
         }
     }
 
-    protected List<ReactPackage> getPackages() {
-        mRnReactNativeGutenbergBridgePackage = new RNReactNativeGutenbergBridgePackage(new GutenbergBridgeJS2Parent() {
-            @Override
-            public void responseHtml(String html, boolean changed) {
-                mContentHtml = html;
-                mContentChanged = changed;
-                mGetContentCountDownLatch.countDown();
-            }
-
-            @Override public void onMediaLibraryPress(MediaSelectedCallback mediaSelectedCallback) {
-                mPendingMediaSelectedCallback = mediaSelectedCallback;
-                onToolbarMediaButtonClicked();
-            }
-        });
-        return Arrays.asList(
-                new MainReactPackage(),
-                new SvgPackage(),
-                new ReactAztecPackage(),
-                new RNRecyclerviewListPackage(),
-                mRnReactNativeGutenbergBridgePackage);
-    }
-
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         View view = inflater.inflate(R.layout.fragment_gutenberg_editor, container, false);
 
         mTitle = view.findViewById(R.id.title);
-        mReactRootView = view.findViewById(R.id.gutenberg);
+        mWPAndroidGlueCode.onCreateView(
+                view.findViewById(R.id.gutenberg),
+                new OnMediaLibraryButtonListener() {
+                    @Override public void onMediaLibraryButtonClick() {
+                        onToolbarMediaButtonClicked();
+                    }
+                },
+                getActivity().getApplication(),
+                BuildConfig.DEBUG,
+                BuildConfig.BUILD_GUTENBERG_FROM_SOURCE);
```

Comment:
Why does this need to be passed through here?

Reply:
> Why does this
Hey James, which one in particular?

Reply:
`BuildConfig.BUILD_GUTENBERG_FROM_SOURCE`

Reply:
Oh, so, the flag is used so we avoid setting ReactInstanceManager's asset bundle file, practically making it mandatory to run the Metro server when in "source" mode. [Relevant code here](https://github.com/wordpress-mobile/gutenberg-mobile/pull/284/files#diff-f2e1dfe800569dfb4ee4830f88779890R99).

I think that having the asset in place can lead to false positives where the developer might miss the fact that Metro is not running (again, when in "source" mode) but the app won't complain because the JS asset will be in place.

Reply:
👍 that makes sense.